### PR TITLE
chore(CI): Adjust testing matrix for Nextcloud 26 on stable26

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Review by come-nc — APPROVED** [review]

File: .github/workflows/static-analysis.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        ocp-version: [ 'dev-master' ]
+        ocp-version: [ 'dev-stable26' ]
```
Comment (ArtificialOwl) [inline .github/workflows/static-analysis.yml:16]: dev-stable26 does not seems to exists in the nextcloud/ocp package, should we keep `dev-master` until we have a 26.x available ?

Comment (susnux) [reply .github/workflows/static-analysis.yml:16]: It is (now) available: https://packagist.org/packages/nextcloud/ocp#dev-stable26

**Comment by nickvergessen** [comment]
Master is 29 in the meantime

**Pull request closed without merging**